src/test/scala/cromwell/engine/WorkflowManagerActorSpec.scala

```diff
@@ -178,5 +178,15 @@ class WorkflowManagerActorSpec extends CromwellTestkitSpec("WorkflowManagerActor
         }
       }
     }
+
+    "run workflows in the correct directory" in {
+      val (wma, workflowId) = runWdl(sampleWdl = SampleWdl.CurrentDirectory,
+                                     EventFilter.info(pattern = s"starting calls: whereami.whereami", occurrences = 1))
+      val outputs = wma.ask(WorkflowManagerActor.WorkflowOutputs(workflowId)).mapTo[binding.WorkflowOutputs].futureValue
+      val outputName = "whereami.whereami.pwd"
+      val salutation = outputs.get(outputName).get
+      val actualOutput = salutation.asInstanceOf[WdlString].value.trim
+      actualOutput.endsWith("/call-whereami") shouldBe true
```

Comment:
Most anything that `shouldBe true` probably has a better ScalaTest matcher. Here, change to `actualOutput should endWith("/call-whereami")`. _<-- Features IntelliJ IDEA removing the space after endWidth_